### DISCUSSION TITLE
Using a backtick in a long sentence goes out of the screen.

### DISCUSSION THREAD

**Discussion by wjrmffldrhrl** [mmistakes/minimal-mistakes#2655] [Q&A] [answered]
I used a backtick (`) in the post and it out of the screen.

# Default size(100%)
![image](https://user-images.githubusercontent.com/49122299/90230336-5b622800-de54-11ea-8bfc-a1aee57c9b2e.png)

# Reduced size(25%)
![image](https://user-images.githubusercontent.com/49122299/90230403-7765c980-de54-11ea-8807-8b8e94d10ce4.png)

like this
~~~text
헤더의 key값은 `Authorization`을 value에는 `Bearer`를 앞에 명시해준 뒤 수신받은 토큰값을 넣어준다. 

ex) `Bearer eyJhbGciOiJIUzUxMiJ9.eyJzdWIiOiJ0ZXN0QG5hdmVyLmNvbSIsImV4cCI6MTU5NzQwMzUxNiwiaWF0IjoxNTk3Mzg1NTE2fQ.eQBXhSVgVebfruLLQbJbi3rm6siyEAStGz349IGcmpvwuWujz5O6OEcgJkIrBBy2aFHHDSI8AKNtmaN4gB6WqQ`
~~~  

This is not a problem, but I leave a post in case it helps.

My Repository and blog url
[blog](https://wjrmffldrhrl.github.io/JWT-with-Spring/)
[Repository](https://github.com/wjrmffldrhrl/wjrmffldrhrl.github.io/blob/master/_posts/2020-08-15-JWT-with-Spring.md)

**Comment by mmistakes** [comment, marked as answer]
That long string above has no spaces which is why it overflows outside of the browser's viewport.
This is the default behavior.

Fixes are:
- Don't use long strings with no spaces that are wider than the smallest screen size you hope to support
- Use multi-line code block instead with triple back tick.
- Override `<code>` CSS to wrap by adding `word-break: break-all` ... this could have unintended issues elsewhere.

![image](https://user-images.githubusercontent.com/1376749/90251740-4661a680-de0c-11ea-8682-d67dfabeddb5.png)

![image](https://user-images.githubusercontent.com/1376749/90251820-698c5600-de0c-11ea-92e6-e88b4f051d31.png)
